Apps/AdminWebClient/src/ClientApp/src/views/Dashboard.vue

```diff
@@ -653,4 +631,16 @@ export default class Dashboard extends Vue {
         padding: 0px;
     }
 }
+
+.refresh-button {
+    position: fixed;
+    right: 16px;
+    top: 80px;
+    z-index: 1000;
+}
+
+h2 {
```

Comment:
instead of applying a style to the heading, how about changing the heading wrapper or adding a class to it to give it more spacing

Reply:
Resolved in latest commit.